src/renderer/components/ReactFlowBox.tsx

```diff
@@ -425,7 +425,7 @@ export const ReactFlowBox = memo(({ wrapperRef, nodeTypes, edgeTypes }: ReactFlo
             >
                 <Background
                     gap={16}
-                    size={0.5}
+                    size={1}
```

Comment:
Why was this changed?

Reply:
Because they changed something internally that made this necessary to get the old size.